requirements.txt

```diff
@@ -1,2 +1,5 @@
-dwave-ocean-sdk>=3.3.0
-matplotlib
+dash==2.14.1
+dash-bootstrap-components==1.5.0
+dwave-ocean-sdk==6.7.1
+Faker==21.0.0
+pandas==2.1.2
```

Comment:
Perhaps loosen up requirements on packages that come preinstalled in the Ocean dev container, like ocean and pandas.